pkgs/development/libraries/science/math/petsc/default.nix

```diff
@@ -30,23 +53,35 @@ stdenv.mkDerivation rec {
     patchShebangs .
     configureFlagsArray=(
       $configureFlagsArray
-      "--CC=$CC"
-      "--with-cxx=$CXX"
-      "--with-fc=$FC"
-      "--with-mpi=0"
+      ${if !mpiSupport then ''
+        "--CC=$CC"
```

Comment:
Is it necessary to set `CC` etc. in the non-mpi case?

Reply:
Seems to be working! The --with-mpi=0 is required though.